src/app/ReviewService.Abstractions/ReviewSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// The review prompt settings used for prompt conditions.
+/// </summary>
+public record ReviewSettings
```

Comment:
I don't think it makes sense to have a record with public setters.
I think you should use either `class` with public setters or a `record` with `init;` properties.

Reply:
Yes, init make more sense, don't know why VS didn't say anything.

Reply:
Don't even know if init is required but by adding it I had to bring a class IsExternalInit for it to work which is odd...

Reply:
That's only because you target netstd2.0.
You wouldn't need that by targeting net6.0

Reply:
Ok let's target .NET 6 then.